### DISCUSSION TITLE
Posting Linkedin Links to Discord

### DISCUSSION THREAD

**Discussion by CBID2** [EddieHubCommunity/support#5822] [Q&A]
Hi @eddiejaoude and co,
One of my LinkedIn connections does this event called SocialSaturdays, and it's where she creates a LinkedIn post at 9am(EST) and users from diverse industries introduce themselves and engage with others in the comment section. Is there a way I can post the link on Discord? I know it's not a GitHub link, but I really feel that it is a great way for our community members to network. 🙂

**Comment by eddiejaoude** [comment]
Sounds interesting. Can you share the link and I can take a closer look?

**Reply by CBID2** [reply]
Here's one from last Saturday @eddiejaoude: https://www.linkedin.com/feed/update/urn:li:activity:6987034502678118400?updateEntityUrn=urn%3Ali%3Afs_feedUpdate%3A%28V2%2Curn%3Ali%3Aactivity%3A6987034502678118400%29

**Reply by eddiejaoude** [reply]
It does look interesting, you can share the link in `#content-share` channel, the bot should allow it - if you could explain a bit about the link I think it will help people understand more about it

**Reply by CBID2** [reply]
Thanks @eddiejaoude. I just posted it! :)
